tools/bin/ecflow/processes/main/meteorological/get_metdata.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+#####get_metdata.py 
+#####usage: <python> <get_metdata.py> <configuration.cfg>
+
+###This script downloads meteorological data through xarray from
+###http://thredds.northwestknowledge.net:8080/thredds/reacch_climate_MET_catalog.html
+###delivered through OPeNDAP.
+
```

Comment:
I know this script is kind of hot mess of endless text. How would a main function help to clean it up?